.github/dependabot.yml

```diff
@@ -0,0 +1,13 @@
+version: 2
+updates:
+- package-ecosystem: gomod
```

Comment:
Copy paste mistake. this is not golang software :-)

Reply:
:hankey: :facepalm: